integrations/api_repo_file_create_test.go

```diff
@@ -28,7 +28,7 @@ func getCreateFileOptions() api.CreateFileOptions {
 		FileOptions: api.FileOptions{
 			BranchName:    "master",
 			NewBranchName: "master",
-			Message:       "Creates new/file.txt",
+			Message:       "Making this new file new/file.txt",
```

Comment:
@richmahn I mean we can use a translated message here. This will not block this PR.

```
Message: ctx.Tr("my language message")
```

Reply:
Why worry about translating a message in the integration test?

Reply:
Unless you were just showing that as an example, but with GitHub there is no translation done with commit messages. But right, not the issue addressed by this PR, as it is just giving the default editor.repo.add, editor.repo.update, editor.repo.delete messages if message is empty.

Reply:
I agree with @richmahn I don't think we should be trying to translate messages in the integration tests.

Reply:
@richmahn  Oh. it's a test file. Ignore my words above.

Reply:
I think a message should never be translated, but as the user intended. Only when it is auto-generated should it be translated. There may be a problem to as what if for some reason they actually wanted to write something like "template" or "home" which would get translated to "Home" or "Template" from our locale_en-US.ini file... while not a big change, it would boggle them why the message is getting uppercased.